src/components/pages/Home.tsx

```diff
@@ -209,20 +260,24 @@ export default function HomePage(): ReactElement {
             <OnboardingSection />
           </section>
         )}
+
         <Container>
-          {queryLatest && (
-            <SectionQueryResult
-              title={
-                hasFeaturedAssets() ? 'Featured Assets' : 'Recently Published'
-              }
-              query={queryLatest}
-              action={
-                <Button style="text" to="/search?sort=created&sortOrder=desc">
-                  All data sets and algorithms →
-                </Button>
-              }
-            />
-          )}
+          <SectionTitle {...featuredAssets.childIndexJson} />
+          {queryLatest?.length > 0 &&
+            queryLatest.map((section) => (
+              <SectionQueryResult
+                key={section.title}
+                title={section.title}
+                query={section.query}
+              />
+            ))}
+          <Button
+            className={styles.allAssetsButton}
+            style="text"
+            to="/search?sort=created&sortOrder=desc"
+          >
+            All data sets and algorithms →
```

Comment:
thought(non-blocking): the Unicode Rightwards Arrow character might not be supported by many fonts. So I would rather use a CSS or SVG icon here.